cmd/avrogen/main_test.go

```diff
@@ -172,3 +172,41 @@ func TestParseTags(t *testing.T) {
 		})
 	}
 }
+
+func TestParseInitialisms(t *testing.T) {
+	tests := []struct {
+		name        string
+		initialisms string
+		wantErr     bool
```

Comment:
```suggestion
		wantErr     assert.ErrorAssertionFunc
```

Reply:
Done, but I named the struct field as `errFunc` to be consistent with its type.